amida-tech/greyscale
# Task/inba 321 new column to task table

What's this PR do?
Created a new column called `isComplete` in the Tasks table
Related JIRA tickets:
[321](https://jira.amida-tech.com/browse/INBA-321)
How should this be manually tested?
Drop your databases and re-run the schema script from `backend/db_setup`:
-`createdb -U indabauser indaba`
-`psql -h localhost -U indabauser indaba < schema.indaba.sql`
- Check that the new Tasks table that was created has `isComplete` as one of the columns
- Run the discussion and task integration test and make sure they both pass
    -`node_modules/.bin/mocha test/discussion.integration.js --bail -t 10000` 
    -`node_modules/.bin/mocha test/task.integration.js --bail -t 10000`

Any background context you want to provide?

Screenshots (if appropriate):
